images/kcp-upgrade/upgrade.sh

```diff
@@ -37,5 +45,6 @@ if [[ "$current_kcp_version" != "$latest_kcp_version" ]]; then
   sed -i "s,$current_kcp_version,$latest_kcp_version,g" images/kcp-registrar/register.sh
   sed -i "s,$current_kcp_version,$latest_kcp_version,g" config/config.yaml
 else
-  exit 1
+  printf "\nNo new kcp version is found, already on latest version.\n"
+  exit 0
```

Comment:
That statement can be removed, the script will exit when quitting the if/else block.